src/v0/destinations/zendesk/transform.js

```diff
@@ -285,21 +295,31 @@ async function getUserId(message, headers, type) {
   }
 }
 
-async function isUserAlreadyAssociated(userId, orgId, headers) {
-  const url = `${endPoint}/users/${userId}/organization_memberships.json`;
+async function isUserAlreadyAssociated(userId, orgId, headers, baseEndpoint) {
+  const url = `${baseEndpoint}/users/${userId}/organization_memberships.json`;
   const config = { headers };
-  const response = await axios.get(url, config);
-  if (
-    response.data &&
-    response.data.organization_memberships.length > 0 &&
-    response.data.organization_memberships[0].organization_id === orgId
-  ) {
-    return true;
+  try {
+    const response = await axios.get(url, config);
+    if (
+      response.data &&
+      response.data.organization_memberships.length > 0 &&
+      response.data.organization_memberships[0].organization_id === orgId
+    ) {
+      return true;
+    }
+  } catch (error) {
+    logger.debug("Error :", error.response ? error.response.data : error);
```

Comment:
Not sure if error will be an object if that is the case we can also see `[object Object]` in logs which is what I'm a little concerned about

May be this'd overcome our earlier problem
```suggestion
logger.debug("Error :")
logger.debug(error?.response?.data || error);
```